### PR TITLE
Fix descriptions in Beatmap Editor (index page)

### PR DESCRIPTION
**Pull request by circleen**
A mouthful of a name, I know.

Also, 'hitsounds stuff.'

### CODE REVIEW

**Comment by circleen** [comment]
Ready for review.

**Comment by cl8n** [comment]
it's assumed that these are ready for review unless if you mention otherwise

**Comment by circleen** [comment]
You can control hitsound volume from the Timing Panel, so I added it in.

File: wiki/Beatmap_Editor/en.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Beatmap Editor
 
-The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
+The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It's separated into four parts:
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/en.md]:
```suggestion
The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
```

File: wiki/Beatmap_Editor/en.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 - [Compose](Compose): The place where a majority of beatmapping happens.
-- [Design](Design): The editor for where Storyboards are made; however, most mappers will only use the "Background Image/Video" button here.
-- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute) and other hitsounds stuff.
+- [Design](Design): The editor for where Storyboards are made; however, most mappers only use the "Background Image/Video" button here.
+- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hitsounds, and the hitsound volume.
 - [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This is sets the values regarding the title, artist, version name, difficulty, combo colours, and more.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/en.md]:
```suggestion
- [Song Setup](Song_Setup): The first place of where all new beatmaps begin. This sets the values regarding the title, artist, version name, difficulty, combo colours, and more.
```

File: wiki/Beatmap_Editor/en.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Beatmap Editor
 
-The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It is separated into four parts:
+The **beatmap editor** is osu!'s in-game graphical interface for creating [beatmaps](/wiki/Beatmaps). It's separated into four parts:
 
 - [Compose](Compose): The place where a majority of beatmapping happens.
-- [Design](Design): The editor for where Storyboards are made; however, most mappers will only use the "Background Image/Video" button here.
-- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute) and other hitsounds stuff.
+- [Design](Design): The editor for where Storyboards are made; however, most mappers only use the "Background Image/Video" button here.
+- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hitsounds, and the hitsound volume.
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor/en.md]:
```suggestion
- [Timing](Timing): The second place of where all new beatmaps begin. This controls the BPM (Beats Per Minute), Timing Points, hit sounds, and the hit sound volume.
```

**Comment by circleen** [comment]
Ready for review now.

**Pull request merged**